prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.html

```diff
@@ -203,90 +204,103 @@
 
 </app-page-section>
 
-<app-page-section *ngIf="oboSites.length || enrolmentErrors?.certificateOrOboSite">
 
-  <app-page-subheader>
-    <ng-container appPageSubheaderTitle>Job Site Information</ng-container>
-
-    <button *ngIf="showEditRedirect"
-            mat-icon-button
-            matTooltip="Edit Job Site Information"
-            (click)="onRoute(EnrolmentRoutes.OBO_SITES)">
-      <mat-icon>edit</mat-icon>
-    </button>
-  </app-page-subheader>
 
-  <ng-container *ngFor="let careSetting of careSettings">
-
-    <ng-container *ngFor="let site of oboSites">
-      <ng-container *ngIf="site.careSettingCode === careSetting.careSettingCode">
-        <app-enrollee-property title="Care Setting"
-                               [makeBold]="true">
-          <div class="mb-3">{{ careSetting.careSettingCode | configCode: 'careSettings' }}
-            <span *ngIf="site.careSettingCode === CareSettingEnum.HEALTH_AUTHORITY">
+<app-page-section>
+  <ng-container
+                *ngIf="oboSites.length || enrolment.jobs.length || enrolmentErrors?.certificateOrOboSite; else legacyJobInfo">
```

Comment:
Does this work as expected?  I would have thought that OR-ing with `enrolment.jobs.length` would mean you'd never reach `legacyJobInfo`

Reply:
Fixed, removed the check for `enrolment.jobs.length`